tests/ScheduledJobsTests/JobHandlers/ParticipantExportJobHandlerTests.cs

```diff
@@ -0,0 +1,147 @@
+    }
+
+    [Test]
+    public async Task HandleAsync_RepositoryThrowsException_LogsErrorAndReturnsFalse()
+    {
+        // Arrange
+        var cancellationToken = new CancellationToken(false);
+        _mockRepository.Setup(repo => repo.GetAllAsync(cancellationToken)).Throws(new Exception("Test exception"));
+
+        // Act
+        var result = await _handler.HandleAsync(new ParticipantExport());
+
+        // Assert
+        Assert.IsFalse(result);
+        _mockLogger.VerifyMessageLogged(LogLevel.Error, "Test exception");
+    }
+
+    private void VerifyAllDependenciesCalledOnce(CancellationToken cancellationToken)
+    {
+        _mockRepository.Verify(repo => repo.GetAllAsync(cancellationToken), Times.Once);
+        _mockCsvUtilities.Verify(
+            csv => csv.WriteCsvToStreamAsync(It.IsAny<IAsyncEnumerable<ParticipantExportModel>>(), It.IsAny<Stream>(), cancellationToken),
+            Times.Once);
+        _mockS3Service.Verify(
+            s3 => s3.SaveStreamContentAsync(
+                _participantExportSettings.S3BucketName,
+                It.Is<string>(fileName => fileName.StartsWith("participant-export-")),
+                It.IsAny<Stream>(),
+                cancellationToken),
+            Times.Once);
+    }
+
+    private IAsyncEnumerable<Participant> GetFakeParticipants()
+    {
+        var participants = new List<Participant>
+        {
+            new()
+            {
+                ParticipantId = "1",
+                Email = "test1@example.com",
+                Firstname = "John",
+                Lastname = "Doe",
+                ConsentRegistration = true,
+                ConsentRegistrationAtUtc = DateTime.UtcNow.AddDays(-10),
+                RemovalOfConsentRegistrationAtUtc = null,
+                MobileNumber = "1234567890",
+                LandlineNumber = "0987654321",
+                Address = new ParticipantAddressModel
+                {
+                    AddressLine1 = "123 Main St",
+                    AddressLine2 = "Apt 4",
+                    AddressLine3 = "Business District",
+                    AddressLine4 = "Suite 500",
+                    Town = "Anytown",
+                    Postcode = "12345"
+                },
+                DateOfBirth = new DateTime(1980, 1, 1),
+                SexRegisteredAtBirth = "M",
+                GenderIsSameAsSexRegisteredAtBirth = true,
+                EthnicGroup = "GroupA",
+                EthnicBackground = "BackgroundA",
+                Disability = false,
+                DisabilityDescription = "None",
+                HealthConditionInterests = new List<string> { "ConditionA", "ConditionB" },
+                CreatedAtUtc = DateTime.UtcNow.AddDays(-20),
+                UpdatedAtUtc = DateTime.UtcNow.AddDays(-5)
+            },
+        };
+
+        return AsyncEnumerable(participants);
+
+        async IAsyncEnumerable<Participant> AsyncEnumerable(IEnumerable<Participant> participants)
```

Comment:
Remove the `async` keyword.

Reply:
removing async causes this error:

`local function 'AsyncEnumerable' with an iterator block must be 'async' to return 'System.Collections.Generic.IAsyncEnumerable<ScheduledJobs.Domain.Participant>'`